system-test/service-path.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it} from 'mocha';
+import * as assert from 'assert';
+import {ServiceError} from 'google-gax';
+import {Bigtable} from '../src';
+import {BigtableClient, BigtableInstanceAdminClient} from '../src/v2';
+
+describe('Service Path', () => {
+  it('Setting universe domain should set the service path', async () => {
```

Comment:
I would suggest also

 1. adding a test for when the universe domain ENV VAR is set.
 2. setting the ENV VAR in the other two tests as well, which will ensure that it's not used when the `universeDomain` and `apiEndpoint` options are set.

Reply:
ok. I'll do that next.

Reply:
Done